src/riak_ql_parser.yrl

```diff
@@ -566,8 +604,36 @@ make_insert({identifier, Table}, Fields, Values) ->
      {values, Values}
     ].
 
-add_limit(A, _B, {integer, C}) ->
-    A#outputs{limit = C}.
+
+%% per Requirements in
+%% https://docs.google.com/document/d/1OUtQwOq2LL3UVGm8mdgPzctpXHV8Dt1SpuX1q6OS9pk:
+%% "If NULLS LAST is specified, null values sort after all non-null
+%%  values; if NULLS FIRST is specified, null values sort before all
+%%  non-null values. If neither is specified, the default behavior is
+%%  NULLS LAST when ASC is specified or implied, and NULLS FIRST when
+%%  DESC is specified (thus, the default is to act as though nulls are
+%%  larger than non-nulls)."
+make_ordby_item(Fld, {asc, _} = Dir, undefined) ->
+    {Fld, Dir, {nulls_last, <<"nulls last">>}};
+make_ordby_item(Fld, {desc, _} = Dir, undefined) ->
+    {Fld, Dir, {nulls_first, <<"nulls first">>}};
+make_ordby_item(Fld, Dir, Nulls) ->
+    {Fld,
```

Comment:
Now there are `make_ord_spec` and `make_null_spec` functions, to lessen eye strain.